react-native-aztec/src/AztecView.js

```diff
@@ -146,7 +146,7 @@ class AztecView extends React.Component {
           onContentSizeChange = { this._onContentSizeChange }
           onHTMLContentWithCursor = { this._onHTMLContentWithCursor }
           onSelectionChange = { this._onSelectionChange }
-          onEnter = { this._onEnter }
+          onEnter = { this.onEnter ? this._onEnter : null }
```

Comment:
Might be really nitpicking but `undefined` is slightly better?

Reply:
I guess you meant `this.props.onEnter && this._onEnter` here?

Reply:
Ah yea!
..and I get used to TS and [only using undefined](https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#null-and-undefined) but found there's no standard equivalent option for eslint..ignore me😅